src/components/Modal/Modal.jsx

```diff
@@ -64,6 +64,7 @@ class Modal extends React.PureComponent {
     isHsApp: PropTypes.bool,
     isOpen: PropTypes.bool,
     icon: PropTypes.string,
+    iconSize: PropTypes.string,
```

Comment:
can we accept string or number and be able to handle either? just a nice to have

Reply:
👍 all done in 2d6cabbf